src/consts/index.ts

```diff
@@ -27,28 +27,42 @@ export enum Language {
   DE = 'de',
   ES = 'es',
   FR = 'fr',
+  HR = 'hr',
+  HU = 'hu',
   HY = 'hy',
   ID = 'id',
   IT = 'it',
   NL = 'nl',
   NO = 'no',
   PL = 'pl',
+  SK = 'sk',
+  SR = 'sr',
+  SV = 'sv',
+  VI = 'vi',
   YO = 'yo',
+  ZH_CN = 'zh_CN',
```

Comment:
`ZH_CN = 'zh-CN'` as in `navigator.language`

Since we are matching the  `navigator.language` value with this `Language` enum